src/main/java/org/sunbird/ratings/exception/ValidateRatingException.java

```diff
@@ -0,0 +1,24 @@
+package org.sunbird.ratings.exception;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
+@ResponseBody
+public class ValidateRatingException extends RuntimeException {
```

Comment:
Can we just have this class as ValidationException and introduce a list of error codes and have a list of messages also. Any API wishing to send a validation error should be able to reuse this. The reason for asking list of messages is because an API may want to perform all validations and give out a list of validation issues

Reply:
These seems good add on.Will do it